tla/consensus/ccfraft.tla

```diff
@@ -929,9 +1030,13 @@ HandleAppendEntriesResponse(i, j, m) ==
            \* "If AppendEntries fails because of log inconsistency: decrement nextIndex (aka sentIndex +1) and retry"
           /\ UNCHANGED matchIndex
     /\ Discard(m)
-    /\ UNCHANGED <<reconfigurationVars, serverVars, candidateVars, logVars, membershipState>>
+    /\ UNCHANGED <<reconfigurationVars, serverVars, candidateVars, logVars>>
 
-\* Any RPC with a newer term causes the recipient to advance its term first.
+\* Any message with a newer term causes the recipient to advance its term first.
+\* Note that UpdateTerm does not discard message m from the set of messages so this 
+\* message can parsed again by receiver. Note that all other message parsing actions should
+\* check that m.term = currentTerm[i] to ensure that this action is not skipped.
```

Comment:
s/skipped/is the only one ENABLED when ?

Reply:
Fixed in https://github.com/microsoft/CCF/pull/5919/commits/e5135f238537660ed9caad1c682b951b4244a236